src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -119,20 +119,25 @@ private static (JObject obj, string path, string name)[] GetDependencies(OpsConf
             {
```

Comment:
use fallback like `docsetConfig?.ECMA2Yaml ?? opsConfig.ECMA2Yaml` to indicate that the semantic is the same regardless of where the config is in?

Reply:
sure